### PR TITLE
SDG 2024 with MolecularNodes Announcement

### PR DESCRIPTION
**Pull request by yuxuanzhuang**
Skeleton for SDG announcement

TODO
- [ ] project description
- [ ] visualization channel
- [ ] invite people to be beta users.

I’ll start writing something today, but please feel free to add anything you think is helpful! @BradyAJohnston

### CODE REVIEW

**Comment by BradyAJohnston** [comment]
Will get some time over the weekend to look at this :)

File: _posts/2024-12-5-sdg_molecularnodes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+layout: post
+title: "NumFocus Small Development Grant: Advancing Molecular Visualization with MolecularNodes"
+---
+
+<img
+src="{{site.images}}/numfocus.png"
+title="NumFOCUS Foundation" alt="NumFOCUS Foundation"
+style="float: right; width: 10em;" />
+
+We are thrilled to announce that we have been awarded a [Small Development Grant][SDG] by NumFocus to enhance scientific molecular rendering with [MolecularNodes][MN] in 2025. This initiative is a collaborative effort between [Yuxuan Zhuang][Yuxuan] and [Brady Johnston][Brady].
```
Comment (orbeckst) [inline _posts/2024-12-5-sdg_molecularnodes.md:11]: I'd replace "we" with "MDAnalysis" to make clearer who "we" is (as the grant is given to the MDAnalysis Org — only NF projects are eligible to receive the grant).

File: public/images/mn_example.png
Comment (orbeckst) [inline public/images/mn_example.png:1]: Awesome teaser!

**Review by orbeckst — COMMENTED** [review]
Great, I am very much looking forward to the results!

If you already have a repo to share, you could add it here, too.

**Comment by orbeckst** [comment]
@yuxuanzhuang and @BradyAJohnston when you're both happy with it, merge it.

**Comment by BradyAJohnston** [comment]
Made some minor changes - mostly switching out `MolecularNodes` -> `Molecular Nodes`. I'm happy if you want to merge it once you double check :)

**Comment by yuxuanzhuang** [comment]
Thanks and merged!

**Pull request merged**